build/docker/zap-baseline.py

```diff
@@ -289,35 +365,50 @@ def main(argv):
 
       pass_count = len(pass_dict)
 
-      # print out the failing rules
+      # print out the ignored rules
       for key, alert_list in sorted(alert_dict.iteritems()):
         if config_dict.has_key(key) and config_dict[key] == 'IGNORE':
-          action = 'IGNORE'
+          user_msg = ''
+          if key in config_msg:
+            user_msg = config_msg[key]
+          print_rule(config_dict[key], alert_list, detailed_output, user_msg)
           ignore_count += 1
-        elif config_dict.has_key(key) and config_dict[key] == 'FAIL':
-          action = 'FAIL'
-          fail_count += 1
-        elif info_unspecified or (config_dict.has_key(key) and config_dict[key] == 'INFO'):
-          action = 'INFO'
+
+      # print out the info rules
+      for key, alert_list in sorted(alert_dict.iteritems()):
+        if config_dict.has_key(key) and config_dict[key] == 'INFO':
+          user_msg = ''
+          if key in config_msg:
+            user_msg = config_msg[key]
+          print_rule(config_dict[key], alert_list, detailed_output, user_msg)
           info_count += 1
-        else:
-          action = 'WARN'
+
+      # print out the warning rules
+      for key, alert_list in sorted(alert_dict.iteritems()):
+        if (not config_dict.has_key(key)) or (config_dict[key] == 'WARN'):
+          user_msg = ''
+          if key in config_msg:
+            user_msg = config_msg[key]
+          print_rule('WARN', alert_list, detailed_output, user_msg)
```

Comment:
It could use `config_dict[key]` instead of the harcoded value like the other loops? Or, a function extracted? (It's the same logic for all, count and print.)